### ISSUE TITLE
Großdeutschland Rework

### ISSUE THREAD

**Issue by clementou** [SighPie/HFM#107]
Three possibilities:
1. Kleindeutschland
   1. Otl, no changes
1. Großdeutschland
   1. Kleindeutschland + Duchy of Austria
   1. Hungarian, Italian, and/or Croation nationalists be winning against Austria in 1848 
      1. Promise Prussia D. Of Austria; war with Hapsburgs; other GPs can intervene
      1. Hapsburgs flee to Budapest, new  decentralized Hapsburg Empire or
      1. Imperial Constituents break off, Hungarian Republic, Italians join Sardinia-Piedmont, D. Of Bohemia German Puppet, Russian Galicia
   1. Vienna Uprising breaks off Austria
       1. King of Prussia accepts Greater German crown from Frankfurt Parliament; liberal constitution
       1. Rest of Austrian Empire does something
   1. Post-Unification rivalry between Prussian Protestants and Austrian Catholics; reflected in modifier; flavor events
1. Mitteleuropa
   1. Großdeutschland + rest of Austrian Empire
   1. Bohemian Germans propose plan to Frankfurt Parliament
   1. Independent/rebelling Hungary represses Transylvanian Saxons; help plea to Frankfurt Parliament
   1. Frankfurt Parliament and Prussian King must accept
   1. War with Austrian Empire; increased likelihood of GP intervention
   1. Ethnically German-dominated; ethnic tension flavor events
   1. Later reformation into Danubian Conferedation-like government

**Comment by mjm2000** [comment]
yes, this would great, 1848 mechanics should be expanded, Maybe a republican unification option with militancy.
